packages/augur-ui/src/modules/filter-sort/components/filter-search.styles.less

```diff
@@ -11,11 +11,13 @@
 
 .Search {
   .text-16;
+  height: @size-40;
 
   align-items: center;
   background: @color-dark-grey;
-  border: none;
-  border-radius: @size-6;
+  border: 1px solid @color-secondary-action;
+  box-sizing: border-box;
+  border-radius: @size-3;
```

Comment:
we actually have a `@border-radius-default` variable that we should use here (it's also 3px) just for future mass adjustments if we need to do them.

Reply:
Updated. Thank you.